CONTRIBUTING.md

```diff
@@ -147,29 +147,61 @@ from the main (upstream) repository:
     git pull --ff upstream master
     ```
     
-## <a name="building-docker"></a>Building a Docker image
-In order to build a Docker image of IBeam, you need to first ensure both Chrome Driver and CPW Gateway are available in `./copy_cache` directory under:
```

Comment:
I'd probably opt for keeping this information as it was - although happy to discuss it if you feel otherwise - as it is essential to completing a build, and may not be straightforward to derive without having it pointed out specifically, as we never mention `/copy_cache` folder and its purpose elsewhere. Also, you reason that:

> I removed the invoke requirements from the repo as the client gateway is already available in the copy_cache directory, and chromium is now installed during Docker builds.

While it's true for the client gateway that it's included in the repo and that chromium is installed during builds, the chrome driver still is provided through copy_cache and its copying can be automated by running the invoke as a pre-build task. Thoughts?

Reply:
Currently the build does not copy any chrome-driver files from the copy_cache (and I don't think it's feasible to have it setup that way due to to possible incompatibilities during multi-platform builds). However replacing the `clientportal.gw` files in the `copy_cache` may be of use, so I guess adding a comment explaining how it is copied during builds may be a good idea. What do you think?

Reply:
You're right! Dang, I missed it in my review of the new Dockerfile, thanks for pointing it out. Where does the chromedriver come from then? Or does IBeam not use it anymore?

Reply:
> DEBIAN_FRONTEND=noninteractive apt-get install -y default-jre dbus-x11 xfonts-base xfonts-100dpi \
        xfonts-75dpi xfonts-cyrillic xfonts-scalable xorg xvfb gtk2-engines-pixbuf nano curl iputils-ping \
        chromium chromium-driver build-essential && \

It is installed using the `apt-get` package manager on [line 26](https://github.com/maksimstojkovic/ibeam/blob/ab4a7f30b5815fcaeeca63fbfab78954bebac162/Dockerfile#L26).

Reply:
right thanks! And don't we need to point to it with env variables? I can see this in the doc for `WebDriver`:

> executable_path - path to the executable. If the default is used it assumes the executable is in the $PATH

I guess we'd need to remove `driver_path` from this instantiation then so that it goes with default:

```
driver = webdriver.Chrome(driver_path, options=options)
```

If you follow that we could deprecate `driver_path` in all functions and logic stack that call `new_chrome_driver`, as well as deprecate the `CHROME_DRIVER_PATH = os.environ.get('IBEAM_CHROME_DRIVER_PATH')`

Right?

Reply:
If you run the following, you can see it's available in the $PATH because you can call it directly without needing to specify the absolute path.

```
docker run --rm voyz/ibeam which chromedriver
```

Reply:
If I'm following correctly, then yes, it should be possible to deprecate those variables because calling `chromedriver` directly from `bash` inside the images runs the `chromium-driver` installed.

Reply:
Right, that's fantastic, this will simplify the code even further. Great job putting it in 👏 Would you like to remove the `driver_path` from code and the `CHROME_DRIVER_PATH` env var? I'm equally well happy to do it myself after we merge this in as I know it may not appear straightforward to you at a glance.

Reply:
I think it's probably best that you handle this removal, I'm only really familiar with the docker build procedures. I would recommend giving the image a test after doing so. Given that multi-platform builds work and all the dependencies get installed correctly, if it works on `amd64`, it should also work the same on `arm64`.

Reply:
Ok, great 👍 I'll take care of it and test it again later. Nice one 👏